### ISSUE TITLE
[Bug]: Migration failed: wrong starting version 1: wrong kv version

### ISSUE THREAD

**Issue by ingoke** [treeverse/lakeFS#6049]
### What happened?

Current Behavior:
After trying to upgrade from v.0.90.1 to v0.102.2, I have executed
lakefs migrate up (with the new version).
        
As a results I've got
Migration failed: wrong starting version 1: wrong kv version

### Expected Behavior

Expected Behavior:
I'd expect that migrate up runs successfully.

### lakeFS Version

v0.102.2

### Deplyoment

on-premise with minio

### Affected Clients

lakectl

### Relevant logs output

```shell
Migration failed: wrong starting version 1: wrong kv version
```

### Contact Details

ingo.kemmerzell@gi-de.com

**Comment by nopcoder** [comment]
@N-o-Z the fix https://github.com/treeverse/lakeFS/pull/6039 probably will probably fix one too

**Comment by N-o-Z** [comment]
@nopcoder Thanks, @ingoke This is fixed in [v0.103.0](https://github.com/treeverse/lakeFS/releases/tag/v0.103.0)

**Comment by ingoke** [comment]
Thank you so much for fixing this issue ad hoc. I really appreciate the immediate response.
